src/content/enterprise/administration/certificates.mdx

```diff
@@ -32,6 +35,9 @@ ingress-nginx:
       default-ssl-certificate: $(POD_NAMESPACE)/your-ssl-certificate-secret
 ```
 
+In order to generate your certificates with [Cert Manager](https://cert-manager.io/) you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
```

Comment:
I checked their official docs, the project is always refered to as `cert-manager` instead of `Cert Manager` so let us also stick with that.

```suggestion
In order to generate your certificates with [cert-manager](https://cert-manager.io/) you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
```